### PR TITLE
MINOR:Updates for new versions and upgrade tests 1.0

### PR DESCRIPTION
**Pull request by bbejeck**
Related to PR https://github.com/apache/kafka/pull/5605 this PR is for system test updates for new versions in branch 1.0

System test ran and passed https://jenkins.confluent.io/job/system-test-kafka-branch-builder/1947/

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
@guozhangwang @mjsax @vvcephei

**Comment by guozhangwang** [comment]
LGTM!

File: bin/kafka-run-class.sh [hunk 1]
```diff
@@ -111,7 +111,7 @@ else
   for file in "$base_dir"/streams/upgrade-system-tests-$SHORT_VERSION_NO_DOTS/build/libs/kafka-streams-upgrade-system-tests*.jar;
   do
     if should_include_file "$file"; then
-      CLASSPATH="$CLASSPATH":"$file"
+      CLASSPATH="$file":"$CLASSPATH"
```
Comment (mjsax) [inline bin/kafka-run-class.sh:114]: why this?

Comment (bbejeck) [reply bin/kafka-run-class.sh:114]: For the same reason in #5605, in enabling the upgrade tests the trunk version of `StreamsUpgradeTest` was executed over the version in `upgrade-system-test` modules. As for why it wasn't a problem before, I can't say.

**Pull request merged**